### PR TITLE
[I18n] CM fixes

### PR DESCRIPTION
**Pull request by soupette**
### What does it do?

This PR fixes some small bugs and add the RBAC management to the locale picker.

### CODE REVIEW

File: packages/strapi-plugin-i18n/admin/src/components/CMEditViewCopyLocale/index.js [hunk 1]
```diff
@@ -54,14 +36,32 @@ const CMEditViewCopyLocale = ({ appLocales, currentLocale, localizations }) => {
...
     try {
+      setIsLoading(true);
       const response = await request(requestURL, { method: 'GET' });
-      const cleanedData = cleanData(response, allLayoutData);
+      const cleanedData = cleanData(response, allLayoutData, localizations);
 
       dispatch({ type: 'ContentManager/CrudReducer/GET_DATA_SUCCEEDED', data: cleanedData });
 
-      handleToggle();
+      strapi.notification.toggle({
```
Comment (soupette) [inline packages/strapi-plugin-i18n/admin/src/components/CMEditViewCopyLocale/index.js:45]: @mfrachet do you want me to split this function?

File: packages/strapi-plugin-i18n/admin/src/components/CMEditViewCopyLocale/index.js [hunk 2]
```diff
@@ -54,14 +36,32 @@ const CMEditViewCopyLocale = ({ appLocales, currentLocale, localizations }) => {
...
+        },
+      });
     } catch (err) {
-      console.log(err);
+      console.error(err);
+
+      strapi.notification.toggle({
+        type: 'warning',
+        message: {
+          id: getTrad('CMEditViewCopyLocale.copy-failure'),
```
Comment (soupette) [inline packages/strapi-plugin-i18n/admin/src/components/CMEditViewCopyLocale/index.js:58]: I may have forgotten to add the translations.

File: packages/strapi-plugin-i18n/admin/src/components/CMEditViewCopyLocale/index.js (hunk 1 above)
Comment (mfrachet) [reply packages/strapi-plugin-i18n/admin/src/components/CMEditViewCopyLocale/index.js:45]: Yes if possible, I was thinking of something like:

```js
const copyLocale = useCopyLocale()
```

and then, you can call the `copyLocale` function inside `handleConfirmCopyLocale` one

File: packages/strapi-plugin-i18n/admin/src/components/CMEditViewCopyLocale/utils/generateOptions.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import get from 'lodash/get';
+
+const generateOptions = (appLocales, currentLocale, localizations, permissions) => {
```
Comment (mfrachet) [inline packages/strapi-plugin-i18n/admin/src/components/CMEditViewCopyLocale/utils/generateOptions.js:3]: Could you please add tests for this one? 🙏🏻

File: packages/strapi-plugin-i18n/admin/src/components/CMEditViewCopyLocale/utils/cleanData.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
+import {
+  contentManagementUtilRemoveFieldsFromData,
+  formatComponentData,
+} from 'strapi-helper-plugin';
+import removePasswordAndRelationsFieldFromData from './removePasswordAndRelationsFieldFromData';
+
+const cleanData = (data, { contentType, components }, initialLocalizations) => {
```
Comment (mfrachet) [inline packages/strapi-plugin-i18n/admin/src/components/CMEditViewCopyLocale/utils/cleanData.js:7]: and also add tests for this one? 🙏🏻

**Pull request merged**